Update meta descriptions for 25 most popular Learn pages (#1370)
astronomer/docs: learn/dynamically-generating-dags.md

---
title: "Dynamically generating DAGs in Airflow"
sidebar_label: "Dynamically generate DAGs"
id: dynamically-generating-dags
---

<head>

>   <meta name="description" content="Get to know the best ways to dynamically generate DAGs in Apache Airflow. See examples of how to generate DAGs using single- and multiple-file methods." />

Reviewer: ```suggestion
  <meta name="description" content="Get to know the best ways to dynamically generate DAGs in Apache Airflow. Use examples to generate DAGs using single- and multiple-file methods." />
```

>   <meta name="og:description" content="Get to know the best ways to dynamically generate DAGs in Apache Airflow. See examples of how to generate DAGs using single- and multiple-file methods." />

Reviewer: ```suggestion
  <meta name="og:description" content="Get to know the best ways to dynamically generate DAGs in Apache Airflow. Use examples to generate DAGs using single- and multiple-file methods." />
```
